exifread/__init__.py

```diff
@@ -122,7 +122,7 @@ def _determine_type(fh: BinaryIO) -> tuple:
 
 def process_file(fh: BinaryIO, stop_tag=DEFAULT_STOP_TAG,
                  details=True, strict=False, debug=False,
-                 truncate_tags=True, auto_seek=True):
+                 truncate_tags=True, auto_seek=True, extract_thumbnail=True):
```

Comment:
Please add documentation in the readme file.

Reply:
Added a note in the README. Thanks for the review.